manifests/r/rammichael/7+TaskbarTweaker/5.14.1/rammichael.7+TaskbarTweaker.locale.en-US.yaml

```diff
@@ -0,0 +1,23 @@
+# Automatically updated by the winget bot at 2022/Dec/23
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: rammichael.7+TaskbarTweaker
+PackageVersion: "5.14.1"
+PackageLocale: en-US
+Publisher: RaMMicHaeL

```

Comment:
```suggestion
Publisher: Ramen Software
```

Reply:
This needs to be moved under https://github.com/microsoft/winget-pkgs/tree/master/manifests/r/RamenSoftware where the publisher added `Windhawk` package. I've created https://github.com/microsoft/winget-pkgs/pull/92295 for PackageId move.